server/datastore/mysql/software.go

```diff
@@ -2322,6 +2323,16 @@ AND EXISTS (SELECT 1 FROM software s JOIN software_cve scve ON scve.software_id
 				SelfService: hs.VPPAppSelfService,
 				IconURL:     hs.VPPAppIconURL,
 			}
+
+			// promote the last install info to the proper destination fields
+			if hs.LastInstallInstallUUID != nil && *hs.LastInstallInstallUUID != "" {
+				hs.AppStoreApp.LastInstall = &fleet.HostSoftwareInstall{
+					InstallUUID: *hs.LastInstallInstallUUID,
```

Comment:
Shouldn't this be renamed `CommandUUID`?

Reply:
Ah yeah, I got confused on the API docs. I'll add a new field.